tests/test_v2.py

```diff
@@ -0,0 +1,94 @@
+import os, sys, pytest
```

Comment:
PEP8, can break this into three statements, I believe pytest is grouped with networkx as outside import